chef/cookbooks/crowbar/templates/default/puma.cfg.erb

```diff
@@ -0,0 +1,50 @@
+ROOT = File.expand_path("../../", __FILE__)
+ENVIRONMENT = ENV["CROWBAR_ENV"] || "production"
+
+THREADS = ENV["CROWBAR_THREADS"] || 1
+WORKERS = ENV["CROWBAR_WORKERS"] || 5
+
+LISTEN = "127.0.0.1"
```

Comment:
I know i said we should set it fixed to 127.0.0.1, but as we have the sysconfig file now it should be cool to have an environment variable for that as well that get's defined within the sysconfig file. Pretty useful for development.